staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -111,17 +107,5 @@ func (f Factory) Create(options config.FormatOptions) (logr.Logger, func()) {
 	// The default is to write to stderr (same as in klog's text output,
 	// doesn't get mixed with normal program output).
 	out := zapcore.Lock(os.Stderr)
-	return NewJSONLogger(out, out)
```

Comment:
This part is clearly a bug. It's the result of a last-minute API change of `NewJSONLogger` from "pass two identical streams to avoid split streams" to "pass nil as second stream".

So passing nil here achieves the stated goal of this commit and PR. However, explaining that in the commit message and PR description would be useful.

Reply:
Will do